pingviewer.pro

```diff
@@ -22,6 +22,15 @@ RESOURCES += \
     QMAKE_LFLAGS += -fopenmp
 }
 
+# Warning as error
+*-g++ | *-clang {
+    QMAKE_CXXFLAGS += -Werror
+}
+*msvc {
+    QMAKE_CXXFLAGS += /WX \
+        /wd4305 \ # Remove truncated warnings, msvc does not provide some non double values
+}
+
```

Comment:
++.
but why are you using qmake for pingviewer?

Reply:
https://github.com/bluerobotics/ping-viewer/issues/549